addon/babel.config.json

```diff
@@ -1,7 +1,16 @@
 {
+  "presets": ["@babel/preset-typescript"],
   "plugins": [
+    [
+      "@babel/plugin-transform-typescript",
+      {
+        "allowDeclareFields": true,
+        "onlyRemoveTypeImports": true,
```

Comment:
onlyRemoveTypeImports can leave side-effecting imports in the output, I don't think we want this?